### PR TITLE
Feature: Code improvements (see log)

### PR DESCRIPTION
**Pull request by brunoocasali**
- Improved the default values according rubocop aspects in gemspec file.
- Added .idea (my IDE) to gitignore to do not add ide files in project.
- Improved controller with `D` note in codeclimate.
- Added syntax highlights in README for some code examples.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +47,7 @@ methods `current_user` and `can_administer?` on your `ApplicationController`
 
 Typical implementation would be:
 
-```ruby
+```rb
```
Comment (iffyuva) [inline README.md]: hi, thanks for the PR. any reason why this is `rb` and not `ruby` ?

**Comment by iffyuva** [comment]
@brunoocasali also is there any reason why all double quotes are converted into single quotes?

File: README.md (hunk 1 above)
Comment (brunoocasali) [inline README.md]: No!! I've just set as default for your README.md (minus is plus! :wink: )

**Comment by brunoocasali** [comment]
Yes, according with rubocop style guide, the default in ruby apps you need to use single quotes in all the time, when you not need to interpolate values on it. Btw I'll open an issue with a propose to keep the gem in rubocop styles see: #66

**Pull request merged**